src/core/sagas/collection.ts

```diff
@@ -0,0 +1,30 @@
+import * as effects from 'redux-saga/effects';
+import FluxCapacitor from '../../flux-capacitor';
+import Actions from '../actions';
+import Adapter from '../adapters/search';
+import Selectors from '../selectors';
+import Store from '../store';
```

Comment:
Nope, I was working from memory. The correct form is indeed `import '../store';`.